src/algo/LCS.js

```diff
@@ -151,21 +158,40 @@ export default class LCS extends Algorithm {
 	reset() {
 		this.oldIDs = [];
 		this.nextIndex = this.initialIndex;
+		this.infoLabelID = this.nextIndex++;
```

Comment:
In this instance, we actually don't need set the infoLabelID on `reset()` so you can delete this line.

When `reset()` gets called none of the IDs prior to the `initialIndex` should be affected, and since in `setup()` you define the `infoLabelID` (line 94) prior to setting the `initalIndex` (line 152), it (along with the code IDs) doesn't have to be reset.